cli/.js/src/main/scala/aqua/ConsoleService.scala

```diff
@@ -0,0 +1,37 @@
+package aqua
+
+import aqua.io.OutputPrinter
+import aqua.js.{CallJsFunction, FluencePeer}
+import aqua.model.func.Call
+import aqua.model.{LiteralModel, VarModel}
+import aqua.model.func.raw.CallServiceTag
+import scala.concurrent.Promise
+
+import scala.scalajs.js.JSON
+
+// Service to print any variables
+class ConsoleService(serviceId: String, fnName: String) {
+
+  def getCallServiceTag(variables: List[VarModel]): CallServiceTag = {
+    CallServiceTag(
+      LiteralModel.quote(serviceId),
+      fnName,
+      Call(variables, Nil)
+    )
+  }
+
+  def registerService(peer: FluencePeer, promise: Promise[Unit]) = {
+    CallJsFunction.registerUnitService(
+      peer,
+      serviceId,
+      fnName,
+      args => {
+        val str = JSON.stringify(args, space = 2)
```

Comment:
is `args` always an array? If so, maybe do something like `args.map(JSON.stringify(_, space = 2)).join(" ")`

This way, you'll get rid of `[ ]` in every log.